src/tests/src/solver/hydro/management/test-hydro-inputs-checker.cpp

```diff
@@ -0,0 +1,112 @@
+//
+// Created by Nikola Ilic on 23/06/23.
+//
+
+#define BOOST_TEST_MODULE hydro - inputs - checker
+#define WIN32_LEAN_AND_MEAN
+#include <boost/test/unit_test.hpp>
+
+#include <antares/study/study.h>
+#include "antares/solver/hydro/management/HydroErrorsCollector.h"
+#include "antares/solver/hydro/management/HydroInputsChecker.h"
+
+#include "fatal-error.h"
+
+using namespace Antares::Solver;
+using namespace Antares::Data;
+
+struct Fixture
```

Comment:
Don't call your struct like this, it creates name collisions that are very difficult to diagnose